src/Raven.Server/ServerWide/Commands/Sharding/PutShardedSubscriptionCommand.cs

```diff
@@ -24,22 +25,51 @@ private PutShardedSubscriptionCommand()
 
     protected override void HandleChangeVectorOnUpdate(ClusterOperationContext context, SubscriptionState existingSubscriptionState, long subscriptionId)
     {
+        var existingShardsCVs = existingSubscriptionState.ShardingState.ChangeVectorForNextBatchStartingPointPerShard;
```

Comment:
can this be `null`?

Reply:
no for sharded subscription existingSubscriptionState.ShardingState is always not null.